src/prebuilt-jupyter-template/.devcontainer.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Prebuilt Workbench JupyterLab devcontainer template",
```

Comment:
just a naming suggestion. 

custom-workbench-jupyter-template

i think custom is a more popular term to describe them.